esp/clients/wsecl/ws_ecl_client.cpp

```diff
@@ -17,12 +17,6 @@
 
 #pragma warning(disable : 4786)
 
-#ifdef _WIN32
```

Comment:
@diegosoaresub deleting this means both client and library is using the same declaration.  I assume we want the library to use "DECL_EXPORT" and the client to use "DECL_IMPORT".

So here perhaps just one line:

#define WS_ECL_CLIENT_API DECL_EXPORT